verification/ci-verify.py

```
@@ -0,0 +1,1555 @@
+            for label, _trigger, _replacement in SANITIZATION_SCENARIOS:
+                record(f"--sanitize {label} [CLI run failed]", False)
+            return
+
+        base_file = _find_file_by_ext(base_dir, ".txt")
+        san_file  = _find_file_by_ext(san_dir, ".txt")
+        if not (base_file and san_file):
+            for label, _trigger, _replacement in SANITIZATION_SCENARIOS:
+                record(f"--sanitize {label} [missing output file]", False)
+            return
+
+        base_text = open(base_file, encoding="utf-8", errors="replace").read()
```

Ruff (0.15.17)

[warning] Use a context manager for opening files

(SIM115)